src/smc-util/db-schema/db-schema.ts

```diff
@@ -174,7 +174,7 @@ schema.blobs = {
   },
   user_query: {
     get: {
-      instead_of_query(database, obj, _account_id, cb) {
+      async instead_of_query(database, obj, cb): Promise<void> {
```

Comment:
removing `_account_id`, which is called as if it is the callback

Reply:
Your PR definitely doesn't fix the bug.   I'm guessing this is totally my fault and was caused by my changing the inputs to instead_of_query, but not realizing it was used here using he older API.   The new api is `instead_of_query(database, opts, cb)`, where opts is 
```
            account_id : undefined
            project_id : undefined
            query      : required
            multi      : required
            options    : required
```
and that's not the same as "obj".   I'll merge your fix, then immediately make the additional changes that are needed.

Reply:
Fortunately, I can't think of anywhere in our codebase that would use this code.  It must be somewhere, but I can't think of it...